www/grpc/config.go

```diff
@@ -1,10 +1,11 @@
 package grpc
 
 type Config struct {
-	Enable       bool          `toml:"enable"`
-	EnableWallet bool          `toml:"enable_wallet"`
-	Listen       string        `toml:"listen"`
-	Gateway      GatewayConfig `toml:"gateway"`
+	Enable              bool          `toml:"enable"`
+	EnableWallet        bool          `toml:"enable_wallet"`
+	Listen              string        `toml:"listen"`
+	BasicAuthCredential string        `toml:"basic_auth_credential"`
```

Comment:
I think we can simply use "Auth" for this config

Reply:
Auth have any method (token base, basic auth, api key and etc...) and we can't use for a method this name, this is not explicit name.